Makefile

```
@@ -50,9 +63,13 @@ else
 	endif # has .git
 	git_stamp ?= 0.0.unknown
 
-    distversion = $(version)-git.$(git_stamp)
-    debrelease = 0git.$(git_stamp)
-    rpmrelease = .git.$(git_stamp)
+    ifneq ($(release_suffix),)
+        distversion = $(release_suffix)$(changed_suffix)
+    else # is not release version
+        distversion = $(version).$(git_stamp)$(changed_suffix)
```

Likely it should be with `-` as we've already discussed

```suggestion
        distversion = $(version)-$(git_stamp)$(changed_suffix)
```